### DISCUSSION TITLE
Add Prediction Guard as a model provider

### DISCUSSION THREAD

**Discussion by sharanshirodkar7** [langgenius/dify#8694] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I’m an AI engineer at Prediction Guard, our API platform is OpenAI API compatible and supports a wide range of state-of-the-art models. You can explore the suite of models we offer [https://docs.predictionguard.com/options/enumerations](url)

Many of our global customers have successfully integrated our platform into their systems, and now they are exploring building low-code and no-code workflows and applications using Dify on top of Prediction Guard. However, adding Prediction Guard as a model provider manually has become a cumbersome process for them.

To streamline this process and improve usability, we kindly request adding Prediction Guard as a model provider directly on the model provider page. This would allow our customers’ end users to access Prediction Guard’s models seamlessly, on the fly, within Dify.

**Comment by dwhitena** [comment]
Thanks @sharanshirodkar7. Note, as a founder at Prediction Guard, I would support @sharanshirodkar7 or another of our team members creating a PR for this directly to Dify (which is awesome by the way)! However, I'm glad that this discussion is started.

P.S. As a co-host of the Practical AI podcast, let me know if we can feature an episode on anything related to Dify. I would love to help get the word out!
